specs/transaction_overview/02_user_interface.rst

```diff
@@ -0,0 +1,90 @@
+=====================
+User interface
+=====================
+
+=====  ====================  =========  ==========
+epic          title           author     created
+=====  ====================  =========  ==========
+`71`_  Transaction overview  tschubotz  2019-01-22
+=====  ====================  =========  ==========
+
+.. _71: https://github.com/gnosis/safe/issues/71
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Transaction list
+------------------
+
+**Sections:**
+
+- Pending: All pending transactions
+- Today: All transactions of today
+- Yesterday: All transactions of yesterday
+- Please put all older transactions in a section that uses the date in the section title.
```

Comment:
IMO in majority cases there will be not more than one transaction per day. Transaction date will be duplicated in two places: section header & transaction cell. Can we use just "Pending", "Today", "Yesterday" and "Past"? Now on iOS we have "Pending", "Today" and "Past" only.

Reply:
> Transaction date will be duplicated in two places: section header & transaction cell. 

The current designs just have the time in the cell. So there is no duplication :)